scripts/genpinctrl/genpinctrl.py

```diff
@@ -507,13 +504,24 @@ def main(cube_path, output):
                         if af["name"] not in entries:
                             entries[af["name"]] = list()
 
+                        # Define the signal mode using, by priority order:
+                        # 1- the config mode (ie: "af["mode"]")
+                        # 2- the inferred mode (an alternate function was found)
+                        if af.get("mode") is not None or family == "STM32F1":
```

Comment:
`if af.get("mode") or family == "STM32F1"` can be used (no need to check explicitely for None)

Reply:
Looks better indeed. Fixed